runtime/doc/undo.txt

```diff
@@ -22,6 +22,16 @@ u			Undo [count] changes.
 :u[ndo] {N}		Jump to after change number {N}.  See |undo-branches|
 			for the meaning of {N}.
 
+:u[ndo]!		Undo one change and remove it from undo history,
+			setting an alternative redo if there is one.
```

Comment:
"setting an alternative redo if there is one". Does this just mean if the user used `C-R` later, other redo branches would be considered? Then I think we can remove this.

Reply:
> "setting an alternative redo if there is one". Does this just mean if the user used `C-R` later, other redo branches would be considered? Then I think we can remove this.

Yes that's what it means. And I'll remove it 👍🏼